plugins/inputs/win_perf_counters/win_perf_counters.go

```diff
@@ -35,6 +35,12 @@ var sampleConfig = `
   #LocalizeWildcardsExpansion = true
   # Period after which counters will be reread from configuration and wildcards in counter paths expanded
   CountersRefreshInterval="1m"
+  ## Accepts a list of PDH error codes which are defined in pdh.go, if this error is encountered it will be ignored
+  ## For example, you can provide "PDH_NO_DATA" to ignore performance counters with no instances
+  ## By default no errors are ignored
+  ## You can find the list here: https://github.com/influxdata/telegraf/blob/master/plugins/inputs/win_perf_counters/pdh.go
+  ## e.g.: IgnoredErrors = ["PDH_NO_DATA"]
+  # IgnoredErrors = []
```

Comment:
I think this needs to be copied into the readme.md too

Reply:
Copied: https://github.com/influxdata/telegraf/blob/forkallowNoDataInConfig/plugins/inputs/win_perf_counters/README.md#ignorederrors